flytekit/loggers.py

```diff
@@ -179,7 +179,7 @@ def get_level_from_cli_verbosity(verbosity: int) -> int:
     :return: logging level
     """
     if verbosity == 0:
-        return logging.CRITICAL
+        return int(os.getenv(LOGGING_ENV_VAR, logging.CRITICAL))
```

Comment:
This is a known bug.
When running the following commands, different behaviors are observed:
- `FLYTE_SDK_LOGGING_LEVEL=40 python wf.py`
- `FLYTE_SDK_LOGGING_LEVEL=40 pyflyte run wf.py wf`
Users should be able to set the environment variables to print the logs when not using `-vv` in the `pyflyte` command.